Cargo.toml

```diff
@@ -179,6 +179,9 @@ exclude = [
     "util/serial",
 ]
 
+[workspace.package]
```

Comment:
if we are making `sgx/` it's own workspace, we also need to remove all the `sgx/` paths from the `members` list above, and we likely need to add `sgx/` to the `exclude` list above

Reply:
hmm, for some reason `sgx` is already in the exclude list. i think i added that to fix a testing issue a long time ago, but i don't remember the details.

Reply:
If you or @nick-mobilecoin can confirm that SGX can be part of the top-level workspace, then I can make the change. If not, I think I'll leave the workspace because I don't want each `sgx` sub-package to hardcode the MSRV.

Reply:
For the sake of making this PR have one concern (update MSRV) I'll remove the the workspace delcaration.